bindings/Sofa/src/SofaPython3/Sofa/Simulation/CMakeLists.txt

```diff
@@ -22,5 +21,5 @@ SP3_add_python_module(
         DESTINATION  Sofa
         SOURCES      ${SOURCE_FILES}
         HEADERS      ${HEADER_FILES}
-        DEPENDS      Sofa.Core Sofa.SimulationCore SofaSimulationGraph SofaPython3::Plugin
+        DEPENDS      Sofa.Simulation.Graph SofaPython3::Plugin
```

Comment:
Should be propagated to [Bindings.SofaConfig.cmake.in](https://github.com/sofa-framework/SofaPython3/blob/master/bindings/Sofa/Bindings.SofaConfig.cmake.in)